Trading/Exchange/binance/tests/tests_sandbox.py

```diff
@@ -0,0 +1,45 @@
+#  Drakkar-Software OctoBot-Tentacles
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import asyncio
+import os
+
+import pytest
+
+import octobot_commons.tests as commons_tests
+import octobot_commons.constants as commons_constants
+import octobot_trading.util.test_tools.spot_rest_exchange_test_tools as spot_rest_exchange_test_tools
+from ...binance import Binance
+
+# All test coroutines will be treated as marked.
+pytestmark = pytest.mark.asyncio
+
+
+async def test_spot_rest():
```

Comment:
It looks like the test is not launched by CI :o

Reply:
I don't know why but it shouldn't before upgrading OctoBot-Trading